wiki/Gameplay/Daily_challenge/fr.md

```diff
@@ -0,0 +1,60 @@
+# Défi quotidien
+
+Le **défi quotidien** est un mode multijoueur de [osu!(lazer)](/wiki/Client/Release_stream/Lazer) où les joueurs peuvent accumuler une série en réussissant à passer des [beatmaps](/wiki/Beatmap) quotidiennement. La difficulté augmente de jour en jour et se réinitialise toutes les semaines.
+
+Chaque beatmap est soigneusement sélectionnée parmi la liste des [Featured Artists](/wiki/People/Featured_Artists), et parfois certaines maps imposent un mod aux joueurs, les obligeant à passer la map avec le mod de jeu sélectionné.
+
+## Menu du jeu
+
+Depuis la page principale du jeu, le menu des défis quotidiens est accessible en suivant les étapes suivantes:
+
+1. Cliquez sur l’icône `jouez` ou appuyez sur `P`.
+2. Cliquez sur l’icône `défi quotidien` ou appuyez sur `D`.
+
+![](img/daily-challenge-menu.png "Capture d'écran du menu du défi quotidien du 7 novembre 2024.")
+
+En entrant, il est présentée au joueur une introduction, montrant la beatmap à passer ainsi que les mods imposés.
+
+Après l'introduction, le joueur consulter les données relatives au défi du jour. Le côté gauche affiche diverses informations en rapport avec le [score](/wiki/Gameplay/Score), comme le nombre total de joueurs ayant réussi la beatmap et le score cumulé de tous les joueurs. Au centre, un classement montre les meilleurs scores obtenus par les joueurs. À droite, les joueurs peuvent discuter du défi quotidien dans un chat.
+
+## Plage de difficulté
+
+![](img/Beatmap-selection-criteria.png "Niveaux d'étoiles spécifiques que les beatmaps doivent respecter pour toute la semaine.")
+
+## Paliers de séries
+
+Les séries de parties sont affichés sur le profil du joueur avec différentes couleurs en fonction de certain paliers atteints:
+
+<!-- tier images: https://www.figma.com/design/tc79qAgJ35KQvdTO0Oj3dN/Daily-Challenge-Counter?node-id=0-1&t=xjRm9Ke0tUMtAQlh-1 -->
+
+|  | Niveau | Participation totale | Série quotidienne | Série hebdomadaire |
+| --: | :-: | :-: | :-: | :-: |
+| ![Lustrous tier](img/tiers/lustrous.png) | Brillant | 1080 jours | 360 jours | 53 semaines |
```

Comment:
I'm pretty sure alt text also needs to be translated, atleast that was done for the Spanish and Polish translations.

Reply:
Yes, it's an oversight of my part, I change that.